openvalidators/reward/open_assistant.py

```diff
@@ -25,7 +26,7 @@ class OpenAssistantRewardModel( BaseRewardModel ):
     reward_model_name: str = "OpenAssistant/reward-model-deberta-v3-large-v2"
 
     @property
-    def name(self) -> str: return "rlhf_reward_model"
+    def name(self) -> str: return RewardModelType.rlhf.value
```

Comment:
Can we rename this model to openassistant?

Reply:
Don't you believe RLHF represents more transparently the type of reward model that is being used?
That way we abstract at the model type level and reduce time of implementation change if we ever want to use other rlhf based model.

Since things move very fast I thought it would be better to be decoupled from the model name and being coupled to framework functionality instead. What do think?

Reply:
Super agree. BTW, @steffencruz @Eugene-hu @opentaco do you know which type of model this dahoas instance is? 
The [model card](https://huggingface.co/Dahoas/gptj-rm-static) on github doesn't tell much

Reply:
I don't know. Ha, the model card doesn't help a bit.

Reply:
Correct me if I am wrong, but aren't both dahoas and relevance trained to act as the reward model in an RLHF system? Just quickly skimming through the model cards, nothing jumps out to me.

Reply:
I don't know about the dahoas model due to lack of information, but it seems that the relevance model is just a BERT (bert-base-uncased) model calculating the RMSE distance for the mean polled embeddings of the completion and the prompt.